package/CHANGELOG

```diff
@@ -2821,3 +2821,7 @@ Testsuite
 
 11/12/07 naveen
   * prepared for release outside lab
+  
+04/02/22 HenokB
+  * change python 2 code snippets to python 3
```

Comment:
Does this entry describe the current PR?

Also, you should read the top of the `CHANGELOG` and look a bit around in order to understand its structure and where you should add this.